pkg/usagestats/seed.go

```diff
@@ -0,0 +1,37 @@
+package usagestats
+
+import (
+	"time"
+
+	jsoniter "github.com/json-iterator/go"
+	prom "github.com/prometheus/prometheus/web/api/v1"
+)
+
+// ClusterSeed identified a unique cluster
+type ClusterSeed struct {
+	UID                    string    `json:"UID"`
+	CreatedAt              time.Time `json:"created_at"`
+	prom.PrometheusVersion `json:"version"`
+}
+
+// JSONCodec works as an interface to encode/decode JSON
+var JSONCodec = jsonCodec{}
+
+type jsonCodec struct{}
+
+// Decode decodes a JSON
+func (jsonCodec) Decode(data []byte) (interface{}, error) {
+	var seed ClusterSeed
+	if err := jsoniter.ConfigFastest.Unmarshal(data, &seed); err != nil {
```

Comment:
Is the use of the `jsoniter` library a performance improvement, or is there a need for it?

Reply:
I took this from Loki, so i'm not sure if it has any benefits using this library. Should we change just to use the standard one?

Reply:
Eh, it's a personal preference to have a good reason for introducing new dependencies, less burden for updating etc. As you wish

Reply:
I just checked and it's an indirect dependency. But you are right, I think i'm gonna change it

Reply:
Yeah we use this in Loki everywhere make sense to not use it here.